Docker build is failing since 82a786a (terser-webpack-plugin Node.js version compatability issue)

The Docker build with the default [Dockerfile](https://github.com/bors-ng/bors-ng/blob/master/Dockerfile) is failing since 82a786a was merged (#967).

The changelog for terser-webpack-plugin [v3.0.0](https://github.com/webpack-contrib/terser-webpack-plugin/releases/tag/v3.0.0) mentions that the minimum supported version of Node.js is now 10.13, while the Node.js version in the Dockerfile is specified as v8.

---

The following is the log from the step where the build fails:

```
> bors-ng-assets@0.0.0 deploy /src
> cd assets && NODE_ENV=dev npm install && npm run deploy

npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN notsup Unsupported engine for terser-webpack-plugin@3.0.2: wanted: {"node":">= 10.13.0"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: terser-webpack-plugin@3.0.2
npm WARN notsup Unsupported engine for cacache@15.0.3: wanted: {"node":">= 10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: cacache@15.0.3
npm WARN notsup Unsupported engine for jest-worker@26.0.0: wanted: {"node":">= 10.14.2"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: jest-worker@26.0.0
npm WARN notsup Unsupported engine for chownr@2.0.0: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: chownr@2.0.0
npm WARN notsup Unsupported engine for mkdirp@1.0.4: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: mkdirp@1.0.4
npm WARN notsup Unsupported engine for move-file@2.0.0: wanted: {"node":">=10.17"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: move-file@2.0.0
npm WARN notsup Unsupported engine for p-map@4.0.0: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: p-map@4.0.0
npm WARN notsup Unsupported engine for tar@6.0.2: wanted: {"node":">= 10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: tar@6.0.2
npm WARN notsup Unsupported engine for chownr@2.0.0: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: chownr@2.0.0
npm WARN notsup Unsupported engine for mkdirp@1.0.4: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: mkdirp@1.0.4
npm WARN notsup Unsupported engine for watchpack-chokidar2@2.0.0: wanted: {"node":"<8.10.0"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: watchpack-chokidar2@2.0.0
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/watchpack-chokidar2/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN assets No description
npm WARN assets license should be a valid SPDX license expression

added 770 packages from 312 contributors and audited 772 packages in 28.288s

34 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> @ deploy /src/assets
> webpack

/src/assets/node_modules/jest-worker/build/WorkerPool.js:25
  } catch {
          ^

SyntaxError: Unexpected token {
    at NativeCompileCache._moduleCompile (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:240:18)
    at Module._compile (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:186:36)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/src/assets/node_modules/jest-worker/build/index.js:18:42)
    at Module._compile (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/src/assets/node_modules/terser-webpack-plugin/dist/index.js:28:42)
    at Module._compile (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/src/assets/node_modules/terser-webpack-plugin/dist/cjs.js:3:16)
    at Module._compile (/src/assets/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ deploy: `webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-05-28T11_36_53_776Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bors-ng-assets@0.0.0 deploy: `cd assets && NODE_ENV=dev npm install && npm run deploy`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bors-ng-assets@0.0.0 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-05-28T11_36_53_810Z-debug.log
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@ RUN apt-get update -q && apt-get --no-install-recommends install -y apt-utils ca
 
 RUN DEBIAN_CODENAME=$(sed -n 's/VERSION=.*(\(.*\)).*/\1/p' /etc/os-release) && \
     curl -q https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
-    echo "deb http://deb.nodesource.com/node_8.x $DEBIAN_CODENAME main" | tee /etc/apt/sources.list.d/nodesource.list && \
+    echo "deb http://deb.nodesource.com/node_12.x $DEBIAN_CODENAME main" | tee /etc/apt/sources.list.d/nodesource.list && \
     apt-get update -q && \
     apt-get --no-install-recommends install -y nodejs
 

